src/lib/Polykey.ts

```diff
@@ -24,17 +28,17 @@ type Metadata = {
 class Polykey {
   polykeyPath: string
   private fs: typeof fs
-  private vaults:Map<string, Vault>
   private metadata: Metadata
   private metadataPath: string
 
   keyManager: KeyManager
-  peerStore: PeerStore
-  peerDiscovery: PeerDiscovery
+  vaults: Map<string, Vault>
```

Comment:
Looks like vaults should have a "manager" type as well?

Reply:
Yeah we can have a vault manager, makes sense from a domain modelling perspective. That's kind of what I was trying to achieve with the `VaultStore` but I think I misinterpreted the need to rename it with a proper name with it being redundant so I took it out completely. Also it's probably more consistent to have a vault manager instead of just a map of vaults :+1:

Reply:
This has been refactored into a new `VaultManager` class that takes care of everything under the `Vault` domain model.